Makefile

```diff
@@ -27,9 +27,18 @@ proto:
 crd:
 	@ cd client && ./hack/update-crd.sh
 
+.PHONY: lint
+# Run golangci-lint
+lint:
+	golangci-lint run
+
 
 # Include release-tools
 
 CMDS=csi-snapshot-metadata
 
 include release-tools/build.make
+
+.PHONY: test
```

Comment:
I don't think we need this line, it's already configured in `build.make` line 18